modules/templates-quickstart.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * assembly/images
+// * openshift_images/using-templates.adoc
 
 [id="templates-quickstart_{context}"]
 = Quickstart templates
```

Comment:
For the next couple of modules, it might be worth trying to make the titles more specific, since it's not very clear which templates you're talking here or that you're talking about ruby on rails for the next couple.